locales/en/index.yml

```diff
@@ -459,6 +459,8 @@ messages:
       fetchCalendars: Error fetching calendars
   unknownSender: Unknown sender
   noContent: Content not available
+  agenda:
+    sectionDate: "dddd D MMMM"
```

Comment:
this date format is not specific of the agenda view, it should be under the global namespace